conf/login.conf

```diff
@@ -0,0 +1,70 @@
+#Topaz login server conf
+
+#--------------------------------
+#login server parametres
+#--------------------------------
+
+#Time-stamp format which will be printed before all messages.
+#Can at most be 20 characters long.
+#Common formats:
+# %I:%M:%S %p (hour:minute:second 12 hour, AM/PM format)
+# %H:%M:%S (hour:minute:second, 24 hour format)
+# %d/%b/%Y (day/Month/year)
+#For full format information, consult the strftime() manual.
+timestamp_format: [%d/%b %H:%M]
+
+#If redirected output contains escape sequences (color codes)
+stdout_with_ansisequence: no
+
+#Makes server output more silent by omitting certain types of messages:
+#Standard     = 1
+#Status       = 2
+#Info         = 4
+#Notice       = 8
+#Warn         = 16
+#Debug        = 32
+#Error        = 64
+#Fatal Error  = 128
+#SQL          = 256
+#Lua          = 512
+#Navmesh      = 1024
+#Action       = 2048
+#Example: "console_silent: 7" Hides standard, status and information messages (1+2+4)
+console_silent: 0
+
+#--------------------------------
+#TCP/IP parameters
+#--------------------------------
+
+login_data_ip: 0.0.0.0
+login_data_port: 54230
+login_view_ip: 0.0.0.0
+login_view_port: 54001
+login_auth_ip: 0.0.0.0
+login_auth_port: 54231
+
+#--------------------------------
+#SQL parameters
+#--------------------------------
+
+mysql_host:      127.0.0.1
+mysql_port:      3306
+mysql_login:     root
+mysql_password:  wolfkin
```

Comment:
yer gonna wanna change yoru password now. also amend / rebase your commits to not include yoru confs